### PR TITLE
Adding argument to allow the user to specify which roc_curve are plotted

### PR DESCRIPTION
**Pull request by doug-friedman**
You tagged this issue #14 as help wanted so I thought I'd pitch in. Feel free to edit if it doesn't match your styIe.

I added a little bit of code to allow the user to pass a list to the roc curve plotting functions to allow them to suppress/show each of the three types of curves: class-specific curves, micro averages, and macro averages.

### CODE REVIEW

**Comment by reiinakano** [comment]
Thanks a lot for taking the time to do this!

Looks good to me so far. Do you mind writing the corresponding unit test as well? Just test out if it works with different permutations of the `curves` argument.

File: scikitplot/classifiers.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def plot_confusion_matrix(clf, X, y, title=None, normalize=False, do_cv=True, cv
 
 
 def plot_roc_curve(clf, X, y, title='ROC Curves', do_cv=True, cv=None,
-                   shuffle=True, random_state=None, ax=None):
+                   shuffle=True, random_state=None, ax=None, curves=['micro','macro','each_class']):
```
Comment (reiinakano) [inline scikitplot/classifiers.py]: stylefix
    ```curves=['micro', 'macro', 'each_class']```

File: scikitplot/plotters.py [hunk 2]
```diff
@@ -188,17 +190,21 @@ def plot_roc_curve(y_true, y_probas, title='ROC Curves', ax=None):
...
-                label='ROC curve of class {0} (area = {1:0.2f})'
-                ''.format(classes[i], roc_auc[i]))
-
-    ax.plot(fpr[micro_key], tpr[micro_key],
-            label='micro-average ROC curve (area = {0:0.2f})'.format(roc_auc[micro_key]),
-            color='deeppink', linestyle=':', linewidth=4)
-    ax.plot(fpr[macro_key], tpr[macro_key],
-            label='macro-average ROC curve (area = {0:0.2f})'.format(roc_auc[macro_key]),
-            color='navy', linestyle=':', linewidth=4)
+    if('each_class' in curves):
```
Comment (reiinakano) [inline scikitplot/plotters.py]: stylefix remove unnecessary parentheses:
```if 'each_class' in curves:```

**Comment by doug-friedman** [comment]
Not a problem! I've added two unit tests for the curve function and updated the branch to reflect your most recent updates.

**Comment by reiinakano** [comment]
Honestly not sure why github changed this to another branch.. 

Perhaps it's best if you submit a new PR

**Comment by doug-friedman** [comment]
Can do

**Pull request closed without merging**